experiments/generic/pod_delete/run_litmus_test.yml

```diff
@@ -44,14 +39,17 @@ spec:
           - name: FORCE
             value: "true"
 
+          - name: LIB
+            value: ""
```

Comment:
what does the var signify? what can be the value here?

Reply:
added comments!